app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    line = command.split()
```

Comment:
You should validate that the `command` string contains exactly three elements separated by spaces and that the first element is 'cp'. This is important to ensure the command is correctly formatted before proceeding with file operations.